docs/README.md

```diff
@@ -958,14 +958,15 @@ The rules for the transformers are defined in a dedicated yaml file, with the fo
 
 ```yaml
 transformations:
-  validation_mode: <validation_mode>
-  table_transformers:
-    - schema: <schema_name>
-      table: <table_name>
-      column_transformers:
-        <column_name>:
-          name: <transformer_name>
-          parameters:
+  validation_mode: <validation_mode> # Validation mode for the transformation rules. Can be one of strict, relaxed or table_level. Defaults to relaxed if not provided
+  table_transformers: # List of table transformations
+    - schema: <schema_name> # Name of the table schema
+      table: <table_name> # Name of the table
+      validation_mode: <validation_mode> # To be used when the global validation_mode is set to `table_level`. Can be one of strict or relaxed. Defaults to relaxed if not provided
```

Comment:
It actually doesn't default to relaxed. If the global one is set to `table_level`, we expect every column to have a validation mode specified, error out otherwise.